app/src/utils/birthday-utils.ts

```diff
@@ -0,0 +1,18 @@
+import { ClaimData } from "../types/claim";
+import moment from "moment";
+import { reformatDate } from "./formatter";
+
+export const check18Plus = (claim: ClaimData): boolean => {
+  if (claim.type === "DateOfBirthCredential") {
+    const userBirthday = reformatDate(claim.value.dob);
+    const age = moment(userBirthday, "YYYYMMDD").fromNow();
+    const userAge = age.split("").slice(0, 2).join("");
+    if (age.includes("in")) {
+      return false;
+    }
+    if (Number(userAge) >= 18) {
+      return true;
+    }
```

Comment:
This method doesn't make sense.  We can use the functions of the moment.js lib to do all this for us.